packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -340,13 +347,15 @@ function getNodeType(node: Member): MemberKind | null {
     case AST_NODE_TYPES.TSConstructSignatureDeclaration:
       return 'constructor';
     case AST_NODE_TYPES.TSAbstractPropertyDefinition:
-      return 'field';
+      return readonly ? 'readonly-field' : 'field';
```

Comment:
When I change this back to just `return 'field'`, all the tests still pass. Looks like there's a gap in coverage here.

(spotted in [Codecov's coverage report](https://app.codecov.io/gh/typescript-eslint/typescript-eslint/pull/6349) that this line is only partially covered)

Reply:
I'll add more tests to capture the cases for readonly `TSAbstractPropertyDefinition` and possibly for `TSPropertySignature` too  (it's also reported partially covered in [Codecov's coverage report](https://app.codecov.io/gh/typescript-eslint/typescript-eslint/pull/6349))